prismic-model/src/exhibition-guides.ts

```diff
@@ -18,13 +18,19 @@ const exhibitionGuides: CustomType = {
       'related-exhibition': link('Related Exhibition', 'document', [
         'exhibitions',
       ]),
+      introTitle: multiLineText({ label: 'Introductory text' }),
```

Comment:
thought: I see from the original ticket that you'll fall back to the exhibition promo if this isn't supplied.

I see you're providing some instructions on how to use the field for the "Standalone title" in the `placeholder` field – is that something we can do here? Or does Prismic return a placeholder we'd then have to filter out later?

Not blocking, just an idea.

Reply:
Good idea. Yep, we can do that. The placeholder text doesn't get returned.